src/NHibernate/Event/Default/DefaultMergeEventListener.cs

```diff
@@ -71,8 +81,19 @@ public virtual void OnMerge(MergeEvent @event)
 							transientEntityName,
 							transientEntity.ToString());
 					}
+
+					// create a comma-delimited list of the entity names that failed to merge
```

Comment:
Darn, it must have been so mind-boggling that I forgot the most obvious thing. :)
Thanks! - Will adjust that...